docs/language/operators/summarize.md

```diff
@@ -5,26 +5,29 @@
 ### Synopsis
 
 ```
-summarize [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>]] [by [<field>][:=<expr>] ...]
+summarize [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>] ...] [by [<field>][:=<expr>] ...]
```

Comment:
I figure the `...` should be added because the user is free to keep tacking on as many aggregate function calls as they want.

Also, I briefly thought about changing `summarize` to `[summarize]` to emphasize its optional nature due to the "implied operator" concept. However, I wasn't sure if we could say it's _always_ optional or if there's times it's necessary to remove ambiguity that I can't think of, so I've left that one as is. But please speak up if you do feel it should be `[summarize]`.

Reply:
@nwt: Ah, thanks for the pointer to precedent! Pushed a commit going with `[summarize]`.